Modules/timemodule.c

```diff
@@ -1282,13 +1365,17 @@ time_process_time(PyObject *self, PyObject *unused)
     return _PyFloat_FromPyTime(t);
 }
 
-PyDoc_STRVAR(process_time_doc,
-"process_time() -> float\n\
-\n\
-Process time for profiling: sum of the kernel and user-space CPU time.");
+/*[clinic input]
+time.process_time_ns
+
+Process time for profiling as nanoseconds:
+
+sum of the kernel and user-space CPU time.
```

Comment:
It is a single sentence. Either write it in one line or make separate sentences.